src/App.js

```diff
@@ -10,7 +15,22 @@ class App extends React.Component {
 
   render() {
     return (
-      <Calculator />
+      <>
+        <Router>
+          {/* <nav>
+            <Link to="/">Home</Link>
+            <Link to="/calculator">Calculator</Link>
+            <Link to="/qoute">Qoute</Link>
+          </nav> */}
+          <NavBar />
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/calculator" element={<Calculator />} />
+            <Route path="/qoute" element={<Qoute />} />
+          </Routes>
+        </Router>
+        {/* <Calculator /> */}
+      </>
     );
```

Comment:
- [ ] Let's try not to have commented codes in our codebase. For any code you're not using in your codebase, it's okay to remove them so your code files look clean and professional. It's a good convention to not have commented codes as they make our codebase dirty. Kindly remove all unused commented codes 👍🏽
 - Lines 20 - 24 
 - Line 32